jfr-daemon/src/main/java/com/newrelic/jfr/daemon/agent/FileJfrRecorder.java

```diff
@@ -19,7 +19,10 @@ public FileJfrRecorder(Recording recording) {
   public Path recordToFile() throws JfrRecorderException {
     try {
       Path output = Files.createTempFile("local-recording", ".jfr");
-      recording.copy(false).dump(output);
+      // creating a copy so we dont stop the original recording while dumping to the file
+      Recording copy = recording.copy(false);
+      copy.dump(output);
+      copy.close();
```

Comment:
Since `jdk.jfr.Recording` is `Closable`, how about:
```suggestion
      try (Recording copy = recording.copy(false)) {
        copy.dump(output);
      }
```